src/lib/migrations/BaseMigrationBuilder.php

```diff
@@ -411,4 +419,31 @@ protected function isNeedUsingExpression(string $fromType, string $toType):bool
         $dates = ['date', 'timestamp'];
         return !(in_array($fromType, $dates) && in_array($toType, $dates));
     }
+
+    public function tmpSaveNewCol(\cebe\yii2openapi\db\ColumnSchema $columnSchema): \yii\db\ColumnSchema
+    {
+        $tableName = 'tmp_table_';
+
+        Yii::$app->db->createCommand('DROP TABLE IF EXISTS '.$tableName)->execute();
+
+        if (is_string($columnSchema->xDbType) && !empty($columnSchema->xDbType)) {
+            $column = [$columnSchema->name.' '.$this->newColStr($columnSchema)];
+        } else {
+            $column = [$columnSchema->name => $this->newColStr($columnSchema)];
+        }
+
+        Yii::$app->db->createCommand()->createTable($tableName, $column)->execute();
```

Comment:
This breaks for me with the following error:

```
Exception 'yii\db\Exception' with message 'SQLSTATE[42000]: Syntax error or access violation: 1067 Invalid default value for 'created_at'
The SQL being executed was: CREATE TABLE `tmp_table_` (
	created_at datetime NOT NULL DEFAULT 'CURRENT_TIMESTAMP'
)'

in /app/vendor/yiisoft/yii2/db/Schema.php:676

Error Info:
Array
(
    [0] => 42000
    [1] => 1067
    [2] => Invalid default value for 'created_at'
)

Caused by: Exception 'PDOException' with message 'SQLSTATE[42000]: Syntax error or access violation: 1067 Invalid default value for 'created_at''

in /app/vendor/yiisoft/yii2/db/Command.php:1302

Stack trace:
#0 /app/vendor/yiisoft/yii2/db/Command.php(1302): PDOStatement->execute()
#1 /app/vendor/yiisoft/yii2/db/Command.php(1102): yii\db\Command->internalExecute('CREATE TABLE `t...')
#2 /app/vendor/cebe/yii2-openapi/src/lib/migrations/BaseMigrationBuilder.php(435): yii\db\Command->execute()
#3 /app/vendor/cebe/yii2-openapi/src/lib/migrations/MysqlMigrationBuilder.php(48): cebe\yii2openapi\lib\migrations\BaseMigrationBuilder->tmpSaveNewCol(Object(cebe\yii2openapi\db\ColumnSchema))
#4 /app/vendor/cebe/yii2-openapi/src/lib/migrations/BaseMigrationBuilder.php(214): cebe\yii2openapi\lib\migrations\MysqlMigrationBuilder->compareColumns(Object(yii\db\mysql\ColumnSchema), Object(cebe\yii2openapi\db\ColumnSchema))
#5 /app/vendor/cebe/yii2-openapi/src/lib/migrations/BaseMigrationBuilder.php(71): cebe\yii2openapi\lib\migrations\BaseMigrationBuilder->buildSecondary()
#6 /app/vendor/cebe/yii2-openapi/src/lib/generators/MigrationsGenerator.php(114): cebe\yii2openapi\lib\migrations\BaseMigrationBuilder->build()
#7 /app/vendor/cebe/yii2-openapi/src/lib/generators/MigrationsGenerator.php(74): cebe\yii2openapi\lib\generators\MigrationsGenerator->buildMigrations()
#8 /app/vendor/cebe/yii2-openapi/src/generator/ApiGenerator.php(485): cebe\yii2openapi\lib\generators\MigrationsGenerator->generate()
#9 /app/vendor/yiisoft/yii2-gii/src/console/GenerateAction.php(53): cebe\yii2openapi\generator\ApiGenerator->generate()
#10 /app/vendor/yiisoft/yii2-gii/src/console/GenerateAction.php(36): yii\gii\console\GenerateAction->generateCode()
#11 [internal function]: yii\gii\console\GenerateAction->run()
#12 /app/vendor/yiisoft/yii2/base/Action.php(93): call_user_func_array(Array, Array)
#13 /app/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\Action->runWithParams(Array)
#14 /app/vendor/yiisoft/yii2/console/Controller.php(182): yii\base\Controller->runAction('api', Array)
#15 /app/vendor/yiisoft/yii2/base/Module.php(552): yii\console\Controller->runAction('api', Array)
#16 /app/vendor/yiisoft/yii2/console/Application.php(180): yii\base\Module->runAction('gii/api', Array)
#17 /app/vendor/yiisoft/yii2/console/Application.php(147): yii\console\Application->runAction('gii/api', Array)
#18 /app/vendor/yiisoft/yii2/base/Application.php(384): yii\console\Application->handleRequest(Object(yii\console\Request))
#19 /app/yii(31): yii\base\Application->run()
#20 {main}
```

column specification is this one:

```
        created_at:
          readOnly: true
          type: string
          format: datetime
          x-db-type: datetime
          nullable: false
```

DB: MariaDB

Reply:
```diff
- default: CURRENT_TIMESTAMP
+ default: current_timestamp()  
```